pkg/ebpf/state.go

```diff
@@ -372,6 +385,27 @@ func (ns *networkState) RemoveConnections(keys []string) {
 	ns.telemetry = telemetry{}
 }
 
+func (ns *networkState) expvarStats() {
+	ticker := time.NewTicker(5 * time.Second)
+	// starts running the body immediately instead waiting for the first tick
+	for ; true; <-ticker.C {
+		ns.Lock()
+		stats := map[string]uint64{
+			"underflows":          ns.telemetry.underflows,
```

Comment:
Now `underflows` are `statsResets`, you should merge the latest master in your branch